### ISSUE TITLE
About using BN layer

### ISSUE THREAD

**Issue by mzy97** [anglixjtu/msg_chn_wacv20#7]
I found that your network not using BN in the entire network. Why you choose this design?

**Comment by anglixjtu** [comment]
@mzy97 According to my experiment, bn does not help improve the accuracy of our network.

**Comment by mzy97** [comment]
@anglixjtu Thank you for your fast reply. Further asking: Will BN harm the accuracy? Or there is no difference in accuracy whether or not add BN. In normal practice, BN will stable training, better convergence, maybe better accuracy when batch size is large. What do you think why bn does not have effect in your network?

**Comment by anglixjtu** [comment]
@mzy97 bn slightly improves the accuracy. But it also slows down the running time. So we do not use it in our network. We also test some other architectures, and bn does work. In fact I'm not sure why bn is of little use to us here, maybe it has something to do with the configuration of the network and hyper-parameters, such as the batch size you mentioned.
